lib/ui/paths.js

```diff
@@ -3,8 +3,6 @@ const { dirname } = require('path');
 // These paths need to be aliased in the manager webpack config to ensure that all
 // code running inside the manager uses the *same* version of react[-dom] that we use.
 module.exports = {
-  '@emotion/core': dirname(require.resolve('@emotion/core/package.json')),
-  '@emotion/styled': dirname(require.resolve('@emotion/styled/package.json')),
```

Comment:
there's also `'emotion-theming': dirname(require.resolve('emotion-theming/package.json')),` below. Does that need to be moved too?

Reply:
You're likely right. I didn't really take time to go through anything outside of `@emotion/core`, but I should check the others as well.